pkg/server/server.go

```diff
@@ -7,26 +7,26 @@ import (
 	"net/http"
 
 	"github.com/gorilla/mux"
-
 	"github.com/spf13/viper"
-
-	"github.com/cmsgov/easi-app/pkg/handlers"
 )
 
-func corsHandler(next http.Handler) http.Handler {
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-
-		w.Header().Set("Access-Control-Allow-Origin", viper.GetString("CLIENT_ADDRESS"))
-		w.Header().Set("Access-Control-Allow-Headers", "Accept, Content-Type, Content-Length, Accept-Encoding, Authorization")
+type server struct {
+	router *mux.Router
+	Config *viper.Viper
+}
 
-		next.ServeHTTP(w, r)
-	})
+func (s *server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	s.router.ServeHTTP(w, r)
 }
 
 // Serve serves all the handlers
 func Serve() {
 	r := mux.NewRouter()
+	s := &server{
+		router: r,
+		Config: viper.New(),
```

Comment:
Can we do `viper.New()` in `/cmd` and pass it to `Serve()` instead of instantiating here?